[2024-01-23T18:43:59.215+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fetch_stackoverflow_data.get_stackoverflow_data scheduled__2022-11-19T00:00:00+00:00 [queued]>
[2024-01-23T18:43:59.312+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fetch_stackoverflow_data.get_stackoverflow_data scheduled__2022-11-19T00:00:00+00:00 [queued]>
[2024-01-23T18:43:59.313+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 6
[2024-01-23T18:43:59.503+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): get_stackoverflow_data> on 2022-11-19 00:00:00+00:00
[2024-01-23T18:43:59.521+0000] {standard_task_runner.py:60} INFO - Started process 4670 to run task
[2024-01-23T18:43:59.561+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'fetch_stackoverflow_data', 'get_stackoverflow_data', 'scheduled__2022-11-19T00:00:00+00:00', '--job-id', '664', '--raw', '--subdir', 'DAGS_FOLDER/loading_data_to_aws_S3.py', '--cfg-path', '/tmp/tmp1139p_7z']
[2024-01-23T18:43:59.601+0000] {standard_task_runner.py:88} INFO - Job 664: Subtask get_stackoverflow_data
[2024-01-23T18:43:59.832+0000] {task_command.py:423} INFO - Running <TaskInstance: fetch_stackoverflow_data.get_stackoverflow_data scheduled__2022-11-19T00:00:00+00:00 [running]> on host 9924936b75b5
[2024-01-23T18:44:00.633+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='charlotte' AIRFLOW_CTX_DAG_ID='fetch_stackoverflow_data' AIRFLOW_CTX_TASK_ID='get_stackoverflow_data' AIRFLOW_CTX_EXECUTION_DATE='2022-11-19T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-11-19T00:00:00+00:00'
[2024-01-23T18:44:01.857+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-01-23T18:44:01.928+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=fetch_stackoverflow_data, task_id=get_stackoverflow_data, execution_date=20221119T000000, start_date=20240123T184359, end_date=20240123T184401
[2024-01-23T18:44:02.418+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-01-23T18:44:02.562+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
